packages-ts/starknet-gauntlet-cli/src/index.ts

```diff
@@ -57,6 +67,25 @@ const registerExecuteCommand = <UI, CI>(
   return registerCommand(deps)
 }
 
+const registerEVMExecuteCommand = <UI, CI>(
+  registerCommand: (deps: EVMDependencies) => EVMCommandCtor<EVMExecuteCommandInstance<UI, CI>>,
+  gauntletConfig,
+) => {
+  const deps: EVMDependencies = {
+    logger: logger,
+    prompt: prompt,
+    makeEnv: (flags) => {
+      return {
+        providerUrl: process.env.NODE_URL || 'https://goerli.infura.io/v3/7c43471f9d604276a856f0cff1edb645',
+        pk: process.env.PRIVATE_KEY,
```

Comment:
Environment var names should probably be different for L1 and L2. The NODE_URL is different, and the private key could be

Reply:
I'm switching between `.env` for now for each command, but yeah, it's not much of a sustainable solution. What do you think if I prepend those with L1/L2 prefix (i.e. L1_PRIVATE_KEY) and continue using one env file?

Reply:
Yep, that seems ok